source/administration/object-management/data-compression.rst

```diff
@@ -0,0 +1,191 @@
+.. _minio-data-compression:
+
+================
+Data Compression
+================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+MinIO Server supports compressing objects to reduce disk usage.
+Objects are compressed before they are written to disk and uncompressed when read, making the compression process transparent to client applications and services.
+
+Depending on the type of data, compression may also increase overall throughput.
+Write throughput for compressible data is generally 500MB per second or greater per available CPU.
+Decompression is typically at least 1 GB per second.
+
+Data compression is a global option, configured settings apply to all buckets in a deployment.
+You can control which objects are compressed by specifying the desired file extensions and media types.
```

Comment:
Re: "media type"

"MIME type" is an outdated but still widely used term. "Media type" is current. Normally I would go with the current term, but the config option is called `mime_types`. Recommendations?

Reply:
We could the ol' fashioned

```
...by specifying the desired file extensions and `media types (MIME) <some link to wikipedia>`__
```